subo/command/deploy.go

```diff
@@ -0,0 +1,70 @@
+package command
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/suborbital/subo/deployer"
+	"github.com/suborbital/subo/project"
+	"github.com/suborbital/subo/subo/util"
+)
+
+var validDeployTypes = map[string]bool{
+	"kubernetes": true,
+	"k8s":        true,
+}
+
+//DeployCmd deploys the current project.
+func DeployCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "deploy",
+		Short: "deploy an application",
+		Long:  "deploy the current project to a remote environment (Kubernetes, etc.)",
```

Comment:
The etc here is misleading I think, because k8s is the only valid place to deploy to at the moment 🙂